src/App.js

```diff
@@ -0,0 +1,59 @@
+import { useState } from 'react';
+import Header from './components/Header';
+import Cards from './components/Cards';
+import Footer from './components/Footer';
+import Instruction from './components/Instruction';
+
+function App() {
+  const [cards, setCards] = useState([]);
+
+  //Delete Card
+  const deleteCard = (card) => {
+    var array = [...cards];
+    var index = array.indexOf(card);
+    if (index !== -1) {
```

Comment:
instead of this you can check at the begining if index is -1 just return from function, to keep code clean and not continue further down the function